ecf/scripts/gdas/atmos/obsproc/dump/jgdas_atmos_tropcy_qc_reloc.ecf

```diff
@@ -1,16 +1,13 @@
-#! /bin/sh
-#BSUB -q %QUEUE%
-#BSUB -P %PROJ%
-#BSUB -J %E%gdas_atmos_tropcy_qc_reloc_%CYC%
-#BSUB -o %COM%/output/%ENVIR%/today/gdas_atmos_tropcy_qc_reloc_%CYC%.o%J
-#BSUB -W 0:05
-#BSUB -n 1
-#BSUB -R affinity[core(1)]
-#BSUB -R span[ptile=1]
-#BSUB -cwd /tmp
+#PBS -S /bin/bash
+#PBS -N %E%gdas_atmos_tropcy_qc_reloc_%CYC%
```

Comment:
We are supposed to remove the `%E%` text from the jobname line:
```
#PBS -N gdas_atmos_tropcy_qc_reloc_%CYC%
```

See the request to remove `%E%` in the NCO checklist for hand-off:
https://docs.google.com/document/d/13p_wvsKTejk7M5q2KMyOeijH4shghlJrTpQlZkThtDI/edit#

Please remove `%E%` from all ecf scripts, thanks!